src/UseCase/BannerSelection/CampaignConfigurationLoader.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\UseCase\BannerSelection;
+
+use FileFetcher\FileFetcher;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * @license GNU GPL v2+
+ */
+class CampaignConfigurationLoader {
+
+	private const SEASONAL_IMPRESSION_LIMIT = 10;
```

Comment:
I'm in favor of making this a global config since Kai mentioned sometimes they need the value to be increased if the data collected is insufficient. It's gonna be :fire: nasty to open a PR just for that.